helm/kubeblocks/falkordb-cluster-omnistrate/tests/integration/replication/test_replication_integration.py

```
@@ -0,0 +1,1922 @@
+                        logger.info(f"Data verified on replica {i+1}")
+                        break  # Success, exit retry loop
+                    elif result.returncode == 2:
+                        # Retry condition (replication not ready yet)
+                        logger.info(f"Replica {i+1} not ready yet, retrying in {retry_delay} seconds...")
+                        logger.info(f"Replica {i+1} output: {result.stdout}")
+                        if retry < max_retries - 1:  # Don't sleep on last retry
+                            time.sleep(retry_delay)
+                            continue
+                        else:
+                            # Last retry failed
+                            assert False, f"Replica {i+1} data not replicated after {max_retries} attempts. Last output: {result.stdout}"
```

Ruff (0.14.8)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)